src/core/installAddon.js

```diff
@@ -333,6 +334,29 @@ export class WithInstallHelpers extends React.Component {
     }
   }
 
+  getCurrentStatus() {
```

Comment:
I believe this method should be named `isAddonEnabled` because we only return the `isEnabled` value.

Reply:
yep i was going to rename this : )